examples/chart/teleport-kube-agent/templates/deployment.yaml

```diff
@@ -65,6 +66,10 @@ spec:
     {{- toYaml .Values.extraVolumeMounts | nindent 8 }}
   {{- end }}
 {{- end }}
+      {{- if .Values.nodeSelector }}
+      nodeSelector:
+        {{- toYaml .Values.nodeSelector | nindent 6 }}
```

Comment:
```suggestion
        {{- toYaml .Values.nodeSelector | nindent 8 }}
```

Reply:
Not sure how I missed that one. Address in 1fa70acd5e5bc3ad7a2422a235fe9ccaff8a9657